requirements-dev.txt

```diff
@@ -3,5 +3,6 @@ pre-commit
 pytest
 pytest-cov
 pytest-mock
+pyupgrade
```

Comment:
no need to specify version ? we use v2.3.0 in the pre commit hooks

Reply:
after changing to 2.4.1 it is working. not sure. anyway removed this from dev reqs